app/controllers/account_reset/cancel_controller.rb

```diff
@@ -6,11 +6,7 @@ def show
       result = AccountReset::ValidateCancelToken.new(token).call
       track_event(result)
 
-      if result.success?
-        handle_valid_token
-      else
-        handle_invalid_token(result)
-      end
+      result.success? ? handle_valid_token : handle_invalid_token(result)
```

Comment:
I have feelings about this.... I really like the visual structure of the if/else and ternaries don't make everything better.... but I can't find any configs for it in the docs :[

Reply:
I can see that but I think in simple cases such as this, ternaries make sense.

Reply:
I was also a bit surprised by how aggressively it converted things to ternaries.